### PR TITLE
Website updates

### PR DESCRIPTION
**Pull request by flovilmart**
Some updates to the main website, long overdues.

section | screenshot
----|----
hero|capture d ecran le 2018-08-16 a 16 51 07
parse-server|capture d ecran le 2018-08-16 a 16 51 15
parse adapters|capture d ecran le 2018-08-16 a 16 51 21

### CODE REVIEW

**Review by natanrolnik — APPROVED** [review]
LGTM!

**Comment by flovilmart** [comment]
Wanted to discuss a bit the tag line and sub header. Not a big copy writer myself.

**Comment by acinader** [comment]
I'm not a fan of focusing on REST server, particularly since we have such nice SDKs.  Focus on REST doesn't really do the whole 'eco-system' justice.  Here's a quick take...

Hit the Ground Running

Parse is a configurable, full stack for quickly deploying modern, full-featured, apps and websites.  Push notifications, user authentication, dashboard, and more right out of the box with dozens of adapters to make integrations easier.

**Review by acinader — APPROVED** [review]

File: css/main.css [hunk 1]
```diff
@@ -963,3 +967,5 @@ footer {
       margin-top: 48px; }
     footer .socialLinks {
       margin-top: 32px; } }
+
+/*# sourceMappingURL=main.css.map */
```
Comment (acinader) [inline css/main.css]: remove?

**Comment by flovilmart** [comment]
I like where you’re going. The sub heading is perhaps a big long?

Painless REST was For the reader to associate with other REST - style server. To serve as an anchor. 

We need a catch line that’s in the middle as ‘hit the ground running’ could be used for anything :)

**Comment by flovilmart** [comment]
what do you think of:

The complete application stack.

Build apps faster with push notifications, user authentication, dashboard and more right out of the box.

File: css/main.css (hunk 1 above)
Comment (flovilmart) [reply css/main.css]: uhm right

**Comment by flovilmart** [comment]
@acinader the 
```
The Parse platform is a configurable, full stack for quickly deploying modern, full-featured, apps and websites. Push notifications, user authentication, dashboard, and more right out of the box with dozens of adapters to make integrations easier.
```
could go above the parse-server / dashboard section? What do you think?

**Comment by acinader** [comment]
@flovilmart 

I like that a lot.  Title case the lead:

The Complete Application Stack

I'd leave the bit as is above the parse-server / dashboard section though.

**Comment by flovilmart** [comment]
I suggested to have 2 bits. 

1 quick 2 liner for the most important features.

THe second, for a more written text about extensibility.

capture d ecran le 2018-08-17 a 09 52 27

File: _includes/header.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <div class="header">
   <div class="container">
     <div class="heroText">
-      <h1>Painless REST backends</h1>
+      <h1>The Complete Application Stack</h1>
       <h2>
           <br/>
-          Build fully featured, <b>data driven</b> applications<br/>
-          with <b>objects</b> and <b>files</b> storage, <b>user authentication</b> and <b>push</b> notifications.
+            Build applications faster with <b>objects</b> and <b>files</b> storage,<br/>
```
Comment (acinader) [inline _includes/header.html]: 'file storage', not 'files storage',  i see that that is how it was, but 'file storage'

Comment (flovilmart) [reply _includes/header.html]: so `object` as well then `object and file storage`

Comment (acinader) [reply _includes/header.html]: yes, which now changes the meaning of the sentence as I was reading it!  ha.  much better.

Comment (flovilmart) [reply _includes/header.html]: So what should it be?

**Review by acinader — APPROVED** [review]

File: _includes/header.html (hunk 2 above)
Comment (acinader) [reply _includes/header.html]: just did a commit to change it

Comment (flovilmart) [reply _includes/header.html]: thanks!

**Comment by flovilmart** [comment]
Awesome! Glad it's finally out!

**Pull request merged**